recipe/meta.yaml

```diff
@@ -1,40 +1,42 @@
-{% set version = "3.4.4" %}
+{% set version = "3.4.5" %}
 
 package:
   name: cryptography
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/cryptography/cryptography-{{ version }}.tar.gz
-  sha256: ee5e19f0856b6fbbdbab15c2787ca65d203801d2d65d0b8de6218f424206c848
+  sha256: 4f6761a82b51fe02cda8f45af1c2f698a10f50003dc9c2572d8a49eda2e6d35b
 
 build:
-  number: 0
+  number: 1
+  skip: True  # [py<36]
   script:
-    - export CRYPTOGRAPHY_DONT_BUILD_RUST=1  # [not win]
-    - set CRYPTOGRAPHY_DONT_BUILD_RUST=1  # [win]
+    - export PYO3_CROSS_INCLUDE_DIR=$PREFIX/include    # [build_platform != target_platform]
+    - export PYO3_CROSS_LIB_DIR=$SP_DIR/../            # [build_platform != target_platform]
+    - export PYO3_CROSS_PYTHON_VERSION=$python         # [build_platform != target_platform]
+    - ls $PYO3_CROSS_LIB_DIR         # [build_platform != target_platform]
+    - find $PYO3_CROSS_LIB_DIR -name "_sysconfigdata*.py" -not -name "${_CONDA_PYTHON_SYSCONFIGDATA_NAME_BACKUP}.py" -type f -exec rm -f {} +  # [build_platform != target_platform]
+    - ls $PYO3_CROSS_LIB_DIR         # [build_platform != target_platform]
```

Comment:
These 3 lines can be removed now because this is done in `cross-python` now.

Reply:
I'll remove them tomorrow. Thanks!